package.json

```diff
@@ -38,12 +39,18 @@
   "devDependencies": {
     "@babel/plugin-transform-modules-commonjs": "^7.18.6",
     "@babel/preset-typescript": "^7.18.6",
-    "@types/jest": "^28.1.5",
+    "@emotion/react": "^11.9.3",
+    "@playwright/experimental-ct-react": "^1.23.3",
+    "@playwright/test": "^1.23.3",
+    "@types/react": "^18.0.15",
     "eslint": "^8.19.0",
     "eslint-config-cheminfo-typescript": "^11.0.1",
-    "jest": "^28.1.3",
     "prettier": "^2.7.1",
     "rimraf": "^3.0.2",
     "typescript": "^4.7.4"
+  },
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
```

Comment:
react is also a dev dependency. It's only used for tests

Reply:
fixed